application/src/main/java/com/sanction/thunder/email/EmailModule.java

```diff
@@ -66,44 +66,66 @@ EmailService provideEmailService(AmazonSimpleEmailService emailService) {
     return new EmailService(emailService, fromAddress);
   }
 
+  @Singleton
+  @Provides
+  MessageOptions provideMessageOptions(@Named("bodyHtml") String bodyHtml,
+                                       @Named("bodyText") String bodyText,
+                                       @Named("successHtml") String successHtml) {
+    if (messageOptionsConfiguration == null) {
+      return new MessageOptions(
+          DEFAULT_SUBJECT, bodyHtml, bodyText, DEFAULT_PLACEHOLDER, successHtml);
```

Comment:
Should we return defaults for just stop emails from happening and give some feedback to the user. I feel it wouldn't be good to email things to a user base with default stuff in it. We should give them a change to correct it before sending emails to people.